### PR TITLE
WebApiHost

### PR DESCRIPTION
**Pull request by glatzert**
Default host for WebApi-Applications based on AspNetCore

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -175,6 +175,7 @@ csharp_style_prefer_method_group_conversion = true:silent
 csharp_style_prefer_top_level_statements = true:silent
 csharp_style_prefer_null_check_over_type_check = true:suggestion
 csharp_style_prefer_local_over_anonymous_function = true:suggestion
+csharp_style_prefer_primary_constructors = true:suggestion
```
Comment (aaron-puhl) [inline .editorconfig:178]: why

File: libraries/JGUZDV.AspNetCore.Logging/src/JGUZDVLoggingExtensions.cs [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
 namespace Microsoft.Extensions.DependencyInjection;
 
+/// <summary>
+/// Extension methods for configuring logging.
+/// </summary>
 public static class JGUZDVLoggingExtensions
 {
-    public static WebApplicationBuilder UseJGUZDVLogging(this WebApplicationBuilder builder, string configSectionName = Constants.DefaultSectionName)
+    /// <summary>
+    /// Configures the web host to use the default logging configuration.
```
Comment (aaron-puhl) [inline libraries/JGUZDV.AspNetCore.Logging/src/JGUZDVLoggingExtensions.cs:11]: default or specified configuration?

File: libraries/JGUZDV.CQRS/src/CQRSDecoratorAttribute.cs [hunk 3]
```diff
@@ -2,7 +2,7 @@
 {
     /// <summary>
     /// Flags a class as a "Decorator".
-    /// Decorators won't be automatically registered as ICommandHandler<> or IQueryHandler<> during DI setup.
+    /// Decorators won't be automatically registered as ICommandHandler&lt;&gt; or IQueryHandler&lt;&gt; during DI setup.
```
Comment (aaron-puhl) [inline libraries/JGUZDV.CQRS/src/CQRSDecoratorAttribute.cs:5]: `<see cref="..." />?`

File: libraries/JGUZDV.Extensions.Authorization/src/ClaimRequirementOptions.cs [hunk 4]
```diff
@@ -3,22 +3,56 @@
...
     public RequirementCollectionMatchType MatchType { get; set; } = RequirementCollectionMatchType.MatchAny;
+
+    /// <summary>
+    /// Gets or sets the requirements in the collection (use with MatchType to create a ClaimRequirementCollection).
+    /// </summary>
     public List<ClaimRequirementOptions>? Requirements { get; set; }
 
 
+    /// <summary>
+    /// Gets or sets a single claim type (use with ClaimValue to create a ClaimValueRequirement).
```
Comment (aaron-puhl) [inline libraries/JGUZDV.Extensions.Authorization/src/ClaimRequirementOptions.cs:23]: see cref?

Comment (aaron-puhl) [reply libraries/JGUZDV.Extensions.Authorization/src/ClaimRequirementOptions.cs:23]: überall?

File: libraries/JGUZDV.WebApiHost/src/Extensions/ConfigurationExtensions.cs [hunk 5]
```diff
@@ -0,0 +1,10 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace JGUZDV.Blazor.WasmServerHost.Extensions
+{
+    internal static class ConfigurationExtensions
+    {
+        public static bool HasConfigSection(this IConfiguration configuration, string configSection)
```
Comment (aaron-puhl) [inline libraries/JGUZDV.WebApiHost/src/Extensions/ConfigurationExtensions.cs:7]: shallow aber ok

**Review by aaron-puhl — COMMENTED** [review]
Warum keine Parameter Summary? :)

File: libraries/JGUZDV.WebApiHost/src/Extensions/ConfigurationExtensions.cs (hunk 5 above)
Comment (glatzert) [reply libraries/JGUZDV.WebApiHost/src/Extensions/ConfigurationExtensions.cs:7]: Wat?

**Pull request merged**